core/workflow-operator/src/main/scala/edu/uci/ics/amber/operator/visualization/nestedTable/NestedTableOpDesc.scala

```diff
@@ -0,0 +1,120 @@
+package edu.uci.ics.amber.operator.visualization.nestedTable
+
+import com.fasterxml.jackson.annotation.{JsonProperty, JsonPropertyDescription}
+import edu.uci.ics.amber.core.tuple.{AttributeType, Schema}
+import edu.uci.ics.amber.operator.PythonOperatorDescriptor
+import edu.uci.ics.amber.operator.metadata.{OperatorGroupConstants, OperatorInfo}
+import edu.uci.ics.amber.core.workflow.OutputPort.OutputMode
+import edu.uci.ics.amber.core.workflow.{InputPort, OutputPort, PortIdentity}
+
+import java.util
+import scala.jdk.CollectionConverters.ListHasAsScala
+
+class NestedTableOpDesc extends PythonOperatorDescriptor {
+
+  @JsonPropertyDescription(
+    "List of columns to include in the nested table chart and their subgroup"
+  )
+  @JsonProperty(value = "add attribute", required = true)
+  var includedColumns: util.List[NestedTableConfig] = _
+
+  override def getOutputSchemas(
+      inputSchemas: Map[PortIdentity, Schema]
+  ): Map[PortIdentity, Schema] = {
+    val outputSchema = Schema()
+      .add("html-content", AttributeType.STRING)
+    Map(operatorInfo.outputPorts.head.id -> outputSchema)
+  }
+
+  override def operatorInfo: OperatorInfo =
+    OperatorInfo(
+      "Nested Table",
+      "Visualize data in a Nested Table",
```

Comment:
Currently the nested level is 1. So please be specific on this in the description